.github/workflows/run_quic_interop.yml

```
@@ -29,11 +29,40 @@ jobs:
         run: |
           jq '.openssl = { image: "quay.io/openssl-ci/openssl-quic-interop"
                          , url: "https://github.com/openssl/openssl"
-                         , role: "client"
+                         , role: "both"
                          }' ./implementations.json > ./implementations.tmp
           mv ./implementations.tmp implementations.json
-      - name: "run interop"
+      - name: "run interop with openssl client"
         run: |
-          python3 ./run.py -c openssl -t ${{ matrix.tests }} -s ${{ matrix.servers }} --log-dir ./logs -d
-
-
+          python3 ./run.py -c openssl -t ${{ matrix.tests }} -s ${{ matrix.servers }} --log-dir ./logs-client -d
+  run_quic_interop_openssl_server:
+    strategy:
+      matrix:
+        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit]
```

```suggestion
        tests: [http3, transfer, handshake, retry, chacha20, resumption, amplificationlimit, ipv6]
```

Shouldn't we include ipv6 test here as well? Also, I don't remember if we meant to add the ipv6 test to `run_quic_interop_openssl_client` as welll.

---

yeah, that was an error, need to add ipv6 back